compiler/src/lnntoamm/Amm.ts

```diff
@@ -114,6 +114,18 @@ export default class Output {
     this.handlers[0] = this.handlers[0].concat(line.concat('\n'));
   }
 
+  call(
+    opcodeName: string,
+    args: string[],
+  ) {
+    let line = this.indent;
+    line = line.concat(opcodeName, '(');
+    line = line.concat(args.join(', '));
+    line = line.concat(')');
```

Comment:
Wondering why this isn't like:

```suggestion
    const line = `${this.indent}${opcodeName}(${args.join(', ')})`;
```

Reply:
makes sense to use!